### PR TITLE
New step to send a Slack message

### PR DESCRIPTION
**Pull request by kinueng**
This is a new step in the pipeline to send a custom Slack message specified in the Environment Variables through the key `message`.

I send the message at the `git-clone` Task to send the alert near the time the initial Slack message send for the pipeline starting.

### CODE REVIEW

File: .tekton/ci/ci-pipeline.yml [hunk 1]
```diff
@@ -361,3 +364,13 @@ spec:
...
+    - name: post-to-slack
+      taskRef:
+        name: slack-post-message
+      runAfter: [git-clone]
+      params:
+        - name: message
+          value: $(params.message)
+      workspaces:
+        - name: workspace
+          workspace: pipeline-ws
```
Comment (aknguyen7) [inline .tekton/ci/ci-pipeline.yml:376]: is this the same as 
image

**Review by aknguyen7 — APPROVED** [review]

**Review by aknguyen7 — COMMENTED** [review]
I'm thinking if we should use the value for message with $(tasks.deploy-application.results.app-url)? we'll have this after `deploy-application` task run

```
params:
        - name: message
          value: $(tasks.deploy-application.results.app-url)
```

**Comment by kinueng** [comment]
OK, that sounds like a much better way to set the URL

**Review by aknguyen7 — APPROVED** [review]
lgtm

**Comment by kinueng** [comment]
Looks like it worked. I made sure to remove the `message` environment variable from this run
image

**Pull request merged**